dist/main.js

```diff
@@ -0,0 +1,2 @@
+/*! For license information please see main.js.LICENSE.txt */
```

Comment:
shouldn't this be excluded from git? I have seen its configured like so. but then it shouldnt be here too

Reply:
yes I added it to gitignore but after the file was already pushed. this was removed here [d2c5c7e](https://github.com/demos-europe/demosplan-ui/pull/13/commits/d2c5c7eb50a67e695834cd1a9f3955378baf726c)